tests/queries_ported/0_stateless/99006_force_drop.sql

```diff
@@ -0,0 +1,5 @@
+CREATE STREAM 99006_v(id int);
+
+INSERT INTO 99006_v(id) values(3);
+
+DROP STREAM 99006_v settings force_drop_big_stream = true;
```

Comment:
This test is meaningless, it's not a big stream.

Reply:
I think it doesn't need auto testing

Reply:
no, this test is only verify this setting work.
or enlarge the ci server size?

Reply:
this is not a big stream, but later if any settings change break, we can find there some issue.
it works more like unit test.